src/components/DetailsForm.js

```diff
@@ -1,22 +1,25 @@
 import React from "react";
 function DetailsForm(props) {
-  const { id, details } = props;
```

Comment:
No need to pass complete details in details form, only pass details which are required in that component, here only single details item shall be passed. @raiab